itests/kit/ensemble.go

```diff
@@ -1084,6 +1085,40 @@ func (n *Ensemble) mirMembership(miners ...*TestMiner) string {
 	return membership
 }
 
+func (n *Ensemble) AddMirValidatorsToFile(fname string, miners ...*TestMiner) {
```

Comment:
Should we de-duplicate these two functions from the ones used in the `cli`? We may want to extract the common code in case we change the format we use to persist the validator set in the future.

Reply:
Fixed